src/Swagger/DocumentationNormalizer.php

```diff
@@ -163,7 +163,7 @@ public function normalize($object, $format = null, array $context = [])
                         if (null !== $path) {
                             $method = $this->operationMethodResolver->getCollectionOperationMethod($resourceClass, $customOperation);
                             $customSwaggerOperation = $this->getSwaggerOperation($resourceClass, $resourceMetadata, $customOperation, [$method], $prefixedShortName, true, $definitions, $method, $object->getMimeTypes());
-
+                            unset($itemOperationsDocs[$resourceClassIri]);
```

Comment:
The code can be made more readable (and more performant) by testing first of a custom operation applies and defaulting to the global one only if no specific is set.

What do you think @Simperfit?

Btw, can you add a non-regression test?